subzero-core/src/main/java/info/jerrinot/subzero/SubZero.java

```diff
@@ -23,22 +23,38 @@ private SubZero() {
      * Use SubZero as a global serializer.
      *
      * This method configures Hazelcast to delegate a class serialization to SubZero when the class
-     * has no explicit strategy configured.
+     * has no explicit strategy configured. Uses {@link Serializer} serializer implementation
+     * internally.
      *
      * @param config Hazelcast configuration to inject SubZero into
      * @return Hazelcast configuration.
      */
     public static Config useAsGlobalSerializer(Config config) {
+        return useAsGlobalSerializerInternal(config, Serializer.class);
+    }
+
+    /**
+     * Use SubZero as a global serializer.
+     *
+     * This method configures Hazelcast to delegate a class serialization to SubZero when the class
+     * has no explicit strategy configured.
+     *
+     * @param config Hazelcast configuration to inject SubZero into
+     * @param serializerClazz Class of global serializer implementation to use
+     * @return Hazelcast configuration.
+     */
+    public static <T> Config useAsGlobalSerializer(Config config, Class<? extends AbstractGlobalUserSerializer> serializerClazz) {
```

Comment:
The way I see it, SubZero is all about providing convenience, so I'm not convinced about not having some convenience method to add custom serialization configuration. The alternative is forcing the user to write out what's already available internally to SubZero class when registering the custom global serializer to hazelcast, which seems a waste.

I couldn't comment on how widely used custom serialization is (probably about 10% of Kryo users going by github stars https://github.com/magro/kryo-serializers) but I will remove the methods if you feel strongly.